velox/common/memory/HashStringAllocator.cpp

```diff
@@ -402,6 +402,7 @@ void HashStringAllocator::checkConsistency() const {
   VELOX_CHECK_EQ(pool_.numLargeAllocations(), 0);
   for (auto i = 0; i < pool_.numSmallAllocations(); ++i) {
     auto allocation = pool_.allocationAt(i);
+    VELOX_CHECK_EQ(allocation->numRuns(), 1);
```

Comment:
If this check is valid, then let's also simplify the loop that follows. That loop runs just once for runIndex of 0.

Reply:
> 

Updated, thanks.